WordPress/Classes/Services/Reader Post/ReaderPostService.swift

```diff
@@ -3,10 +3,6 @@ import WordPressKit
 
 extension ReaderPostService {
```

Comment:
Hi @salimbraksa, I noticed you just added this extension in #20193, just an FYI to let you know that the code has been slightly changed. I don't want you to be suprised why the code you just added suddenly looks different after this PR is merged. 😄 

A quick summary of what are the changes, basically the `ReaderPostService.managedObjectContext` is replaced with a `CoreDataStack` instance, and the code where performs Core Data read and write need to be changed along with it.